lecli/api_utils.py

```diff
@@ -285,3 +294,45 @@ def gensignature(api_key, date, content_type, request_method, query_path, reques
     digest.update(canonical_string)
 
     return digest.digest()
+
+
+def get_management_url():
+    """
+    Get management url from the config file
+    """
+    config_key = 'management_url'
+    try:
+        url = CONFIG.get(URL_SECTION, config_key)
+        if validators.url(str(url)):
+            return url
+        else:
+            print_config_error_and_exit(URL_SECTION, 'Log Management URL(%s)' % config_key)
+    except ConfigParser.NoOptionError:
+        print_config_error_and_exit(URL_SECTION, 'Log Management URL(%s)' % config_key)
+
+
+def pretty_print_string_as_json(text):
+    """
+    Pretty prints a json string
+    """
+    print json.dumps(json.loads(text), indent=4, sort_keys=True)
```

Comment:
we are using tabular in other api's for the moment: https://pypi.python.org/pypi/tabulate
but you can consider using this library as well: https://pypi.python.org/pypi/PTable/0.9.0